SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -2628,6 +2662,12 @@ void USpatialReceiver::MoveMappedObjectToUnmapped(const FUnrealObjectRef& Ref)
 	}
 }
 
+void USpatialReceiver::RetireWhenAuthoritive(Worker_EntityId EntityId, Worker_ComponentId ActorClassId, bool bIsNetStartup, bool bNeedsTearOff)
+{
+	DeferredRetire DeferredObj = { EntityId, /*ActorClassId*/ ActorClassId, /*bIsNetStartupActor*/ bIsNetStartup, /*bNeedsTearOff*/bNeedsTearOff };
```

Comment:
The variable names we pass in are the same/very similar to the field names, so the comments are redundant.